app/database/schema.prisma

```diff
@@ -498,18 +503,52 @@ model Invite {
   updatedAt DateTime @updatedAt
 }
 
-
 model Announcement {
   id String @id @default(cuid())
-  
-  name   String
+
+  name    String
   content String
 
-  link String?
-  linkText String?
+  link      String?
+  linkText  String?
   published Boolean @default(false)
 
   // Datetime
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 }
+
+enum BookingStatuses {
```

Comment:
Can you please change the name to `BookingStatus`. We have all our models and enums in singular rather than plural.